sdk/eventgrid/azure-eventgrid/azure/eventgrid/_helpers.py

```diff
@@ -134,3 +136,27 @@ def _cloud_event_to_generated(cloud_event, **kwargs):
         additional_properties=cloud_event.extensions,
         **kwargs
     )
+
+def _build_request(endpoint, content_type, events, client):
+    serialize = client._serialize # pylint: disable=protected-access
```

Comment:
you don't actually really needs this. For the meantime, you can directly import `Serializer` from msrest and set `serialize = Serializer()`.

This will get rido f the need to pass the `client` parameter to `_build_request`